source/apiVolontaria/apiVolontaria/serializers.py

```diff
@@ -111,6 +113,23 @@ class Meta:
         validators=[phone_number],
     )
 
+    managed_cell = serializers.SerializerMethodField()
+
+    def get_managed_cell(self, obj):
+        cells = Cell.objects.filter(managers__in=[obj])
+
+        array_cells = list()
+
+        for cell in cells:
+            data = {
+                'id': cell.pk,
+                'name': cell.name,
+            }
```

Comment:
Here we can use a NestedSerializer in place of a hardcoded structure so it's more easy to refactor in the futur. You can use the `CellBasicSerializer` that contain data available for everybody connected :)

Reply:
I have tried but it's kinda weird to have the managers section in the cell : 

{
    "id": 1,
    "username": "yolivier",
    "email": "yanic.olivier@gmail.com",
    "first_name": "Yanic",
    "last_name": "Olivier",
    "is_active": true,
    "is_superuser": true,
    "phone": null,
    "mobile": null,
    "managed_cell": [
        {
            "id": 1,
            "name": "Montréal",
            "address": {
                "id": 1,
                "address_line1": "5333 avenue Casgrain",
                "address_line2": "local 201",
                "postal_code": "H2T 1X3",
                "city": "Montréal",
                "country": {
                    "iso_code": "CA",
                    "name": "Canada"
                },
                "state_province": {
                    "iso_code": "QC",
                    "name": "Québec"
                }
            },
            "managers": [
                {
                    "id": 1,
                    "username": "yolivier",
                    "first_name": "Yanic",
                    "last_name": "Olivier",
                    "email": "yanic.olivier@gmail.com"
                }
            ]
        }
    ]
}

Reply:
Yeah it's true :p
But can be a good things when we have multiple managers and it's the default représentation of a cell.